workspace/utilities/com/module-embed.xsl

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- 
+	Allows embedding of multiple sources throught oembed standard.
```

Comment:
@dominic-blain Can we get info here on what needs to be imported in xslt in order to make it work ?

Reply:
Module-embed imports all xsl files it needs. So if you import module-embed in site-import (and add js files specified in the comment), it will work.

Reply:
@nitriques Ping

Reply:
> So if you import module-embed in site-import

Exact. That what I want to copy paste (just like I can copy paste the js ones...